core/commands/commands_test.go

```diff
@@ -224,6 +224,7 @@ func TestCommands(t *testing.T) {
 		"/storage",
 		"/storage/path",
 		"/storage/upload",
+		"/storage/filelist",
```

Comment:
suggest renaming to `/storage/files/list` to match other similar ones, so you can implement other `/storage/files/` functions later